### PR TITLE
Add Hebrew localization

### PR DESCRIPTION
**Pull request by yuvalabou**

### CODE REVIEW

**Comment by AitorDB** [comment]
Thanks for the PR! There's already another PR for Hebrew localization, I can see that there's a couple of differences in the translations, would you mind to take a look to it? https://github.com/AitorDB/home-assistant-sun-card/pull/44

**Review by baget — APPROVED** [review]

**Review by baget — APPROVED** [review]

**Review by baget — APPROVED** [review]

**Comment by yuvalabou** [comment]
@AitorDB Can we please merge this as @baget approved my changes?

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Note: If you get an error similar to this `Custom element doesn't exist` you wil
 | timeFormat    | `'12h'`/`'24h'`      | Displayed time format                | Locale based on Home assistant language             |
 | title         | `string`             | Card title                           | Doesn't display a title by default                  |
 
-(<sup>1</sup>) Supported languages: `da`, `de`, `en`, `es`, `et`, `fi`, `fr`, `hu`, `it`, `nl`, `pl`, `pt-BR`, `ru`, `sk`, `sl`, `sv`
+(<sup>1</sup>) Supported languages: `da`, `de`, `en`, `es`, `et`, `fi`, `fr`, `he`, `hu`, `it`, `nl`, `pl`, `pt-BR`, `ru`, `sl`, `sv`
```
Comment (AitorDB) [inline README.md]:
```suggestion
(<sup>1</sup>) Supported languages: `da`, `de`, `en`, `es`, `et`, `fi`, `fr`, `he`, `hu`, `it`, `nl`, `pl`, `pt-BR`, `ru`, `sk`, `sl`, `sv`
```

File: src/constants.ts [hunk 2]
```diff
@@ -34,7 +35,7 @@ export class Constants {
 
   static readonly HORIZON_Y = 108
   static readonly LOCALIZATION_LANGUAGES: Record<string, TSunCardTexts> = {
-    da, de, en, es, et, fi, fr, hu, it, nl, pl, 'pt-BR': ptBR, ru, sk, sl, sv
+    da, de, en, es, et, fi, fr, he, hu, it, nl, pl, 'pt-BR': ptBR, ru, sl, sv
```
Comment (AitorDB) [inline src/constants.ts]:
```suggestion
    da, de, en, es, et, fi, fr, he, hu, it, nl, pl, 'pt-BR': ptBR, ru, sk, sl, sv
```

**Pull request merged**